src/go/rpk/pkg/cli/cmd/common/common_test.go

```diff
@@ -317,19 +309,35 @@ func TestKafkaAuthConfig(t *testing.T) {
 	}
 }
 
+type envVar struct {
```

Comment:
Since this type is only used in `TestBuildTLSConfig`, can it move into the test function just below?

Reply:
Also, looking at it further, the test expects exactly three env vars, and expects the three env vars to be in order of the exact keys `RP_ADMIN_CERT_FILE`, `RP_ADMIN_KEY_FILE`, and `RP_ADMIN_TRUSTSTORE_FILE`.

I think this type can be replaced with something like,
```go
type envCertVars {
  keyFile string
  certFile string
  caFile string
}
```
and `envvars` in the test struct below can have field type `*envCertVars`? This would simplify definition in the tests and would simplify usage in running the tests.

Reply:
Sounds good.

Reply:
I ended up removing it while making further changes.